CMCL-1660: Problems with samples (#1050)
Unity-Technologies/com.unity.cinemachine: com.unity.cinemachine/CHANGELOG.md

- Regression fix: Confiner2D was not always confining when a camera was newly activated.
- The RotationComposer no longer damps in response to composition changes from the FreeLookModifier.
- The game-view composer guides dynamically reflect the current composition when a FreeLookModifier is changing it.

> - Sample assets are correctly selected at import time depending on current render pipeline and input configuration.  Re-importing samples should fix any existing issues.

Reviewer: ```suggestion
- Fixed sample asset selection at import depending on current render pipeline and input configuration.  You need to re-import the samples to fix any existing issues.
```
